cmd/bootstrap/cmd/machine_account_key.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd/bootstrap/run"
+	"github.com/onflow/flow-go/model/bootstrap"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	ioutils "github.com/onflow/flow-go/utils/io"
+)
+
+var (
+	flagNodeID string
+)
+
+// machineAccountKeyCmd represents the `machine-account-key` command which generates required machine account key
+// and writes it to the default path within the bootstrap directory. Used by existing operators to create the
+// machine account key only
+var machineAccountKeyCmd = &cobra.Command{
+	Use:   "machine-account-key",
+	Short: "Generates machine account key and writes it to the default path within the bootstrap directory",
+	Run:   machineAccountKeyRun,
+}
+
+func init() {
+	rootCmd.AddCommand(machineAccountKeyCmd)
+
+	machineAccountKeyCmd.Flags().BytesHexVar(&flagMachineSeed, "seed", generateRandomSeed(), fmt.Sprintf("hex encoded machine account seed (min %v bytes)", minSeedBytes))
+}
+
+// machineAccountKeyRun generate a machine account key and writes it to a default file path.
+func machineAccountKeyRun(_ *cobra.Command, _ []string) {
```

Comment:
Please add a test for this command following the [example from `key`](https://github.com/onflow/flow-go/pull/722/files#diff-fd992efa53b7e741bbf99ff01fce62a02af4798fc9c5dd075489d461efc6e7dcR42).

We should:
* test that the machine account key file is created when we run the command
* test that an existing machine account key file is not overwritten when we run the command where a file already exists